test/controllers/email/VerifyEmailControllerSpec.scala

```diff
@@ -75,583 +71,97 @@ class VerifyEmailControllerSpec extends ControllerBaseSpec with MockEmailVerific
     }
   }
 
```

Comment:
Could you add back in a little unit test for the `emailSendVerification` action. Simply to verify that it results in a redirect status (303) and redirects to the expected place